### PR TITLE
Add github organization access note

### PR DESCRIPTION
**Pull request by liggitt**
Related to https://bugzilla.redhat.com/show_bug.cgi?id=1308843

### CODE REVIEW

File: install_config/configuring_authentication.adoc [hunk 1]
```diff
@@ -884,7 +884,10 @@ application]. The application must be configured with a callback URL of
 `<master>/oauth2callback/<identityProviderName>`.
 <6> The client secret issued by GitHub.
 <7> Optional list of organizations. If specified, only GitHub users that are members of 
-at least one of the listed organizations will be allowed to log in.
+at least one of the listed organizations will be allowed to log in. If the GitHub OAuth
+application configured in *client_id* is not owned by the organization, an organization
```
Comment (adellape) [inline install_config/configuring_authentication.adoc]: Should this be

```
`*clientID`*
```

as shown in `<5>`?

Comment (liggitt) [inline install_config/configuring_authentication.adoc]: yes

**Comment by adellape** [comment]
:+1:

**Comment by adellape** [comment]
**revhistory note:**

Install & Config -> Configuring Authentication

```
Added GitHub organization access note to the link:../install_config/configuring_authentication.html#GitHub[GitHub] section.
```

**Pull request merged**